float32[] joint_torques

# Normal at each toe
float32[] normal_toe

# Force at each toe 
float32[] toes_force1 
float32[] toes_force2 

# Ground friction coefficients at each toe 
float32[] ground_friction 

# Height scan at each toe 
float32[] height_scan
